process/smartContract/scQueryService.go

```diff
@@ -91,6 +101,21 @@ func (service *SCQueryService) ExecuteQuery(query *process.SCQuery) (*vmcommon.V
 	return service.executeScCall(query, 0)
 }
 
+func (service *SCQueryService) shouldAllowQueriesExecution() bool {
+	// TODO: analyze if we need this check. this was added because of the TestExecuteQuery_EmptyFunctionShouldErr test that
```

Comment:
Please remove this along with the following lines from scQueryService_test.go
L120 & L139
All tests will pass